erts/doc/src/erl_nif.xml

```diff
@@ -2770,6 +2784,8 @@ enif_map_iterator_destroy(env, &amp;iter);</code>
           a mutable copy is allocated and assigned to <c>*bin</c>.</p>
         <p>Returns <c>true</c> on success, or <c>false</c> if memory allocation
           failed.</p>
+        <p>The returned pointer is suitably aligned for any built-in type that
+          fit in the allocated memory.</p>
```

Comment:
Wrong place. This is enif_realloc_binary docs.

Reply:
Ugh, fixed.